### ISSUE TITLE
Aberrant radial artery

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11653]
# Aberrant radial artery

## Proposed definition

A radial artery that follows an atypical anatomical course or occupies an abnormal position.

## Parent term

- `HP:0031640 Abnormal radial artery morphology`

## Synonyms

- Displaced radial artery

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

**Comment by pnrobinson** [comment]
@evanc1648
@mellybelly 

This LABEL does not add much above and beyond Abnormal radial artery morphology, but I think that the term definition implies
one of the types of anomalous radial artery, viz.,


High-origin (brachioradial) 
Superficial radial artery 
RA tortuosity
RA Duplication
Variant branching
Hypoplasia/Aplasia

I did not see much about association with Fanconi anemia for any of these, but there was one article:
https://pubmed.ncbi.nlm.nih.gov/11104997/

I would suggest if we want specific terms that is fine, otherwise we should just use the parent term.

**Comment by evanc1648** [comment]
Thank you, @pnrobinson! 

Here's the full provenance for this term. This traces to the Fanconi Anemia Clinical Care Guidelines ([Chapter 2](https://fanconi.org/clinical-care-guidelines/diagnosis-of-fanconi-anemia-testing-and-genetic-counseling/)), whose table cites De Kerviler et al. ([PMID:10816398](https://pubmed.ncbi.nlm.nih.gov/10816398/)). The quote from the citation is: "The peripheral vessels may also be involved, the most common abnormality being an **absent or displaced radial artery** (45% of patients)."

The source finding is "absent or displaced radial artery." We de-bundled it ("absent" or "displaced"), where this request is for "displaced" ("absent" maps to the existing Radial artery aplasia, HP:0020118). You're right that our label "Aberrant radial artery" was too broad and essentially duplicated the parent term.

The proposed term would be:
- Label: Displaced radial artery
- Definition: An abnormal position of the radial artery, displaced from its normal anatomical location. 
- Parent: HP:0031640 Abnormal radial artery morphology

This term is about the position/location of the radial artery, which may be a distinct feature that warrants a new term. If you feel that it's too general and would prefer we annotate the parent, we're happy to.

**Comment by pnrobinson** [comment]
@evanc1648  thanks for the clarification!
Adding new term: 
- ID: HP:6001532
- label: Displaced radial artery
- parents: [Abnormal radial artery morphology (HP:0031640)](https://hpo.jax.org/app/browse/term/HP:0031640)
- pmid: PMID:10816398
- def: An abnormal position of the radial artery, displaced from its normal anatomical location.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11653
- synonyms:
